scripts/insecure_grpcurl_series.sh

```diff
@@ -38,7 +38,7 @@ if [ -z "${REQUESTED_MAX_TIME}" ]; then
   exit 1
 fi
 
-go install github.com/fullstorydev/grpcurl/cmd/grpcurl
```

Comment:
We don't like taking latest as there is high chances it won't work the same and did not break compatibility 🙃 Can we pin to latest version instead? (or some version)

Reply:
Sure thing, done :+1: